packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -68,32 +74,35 @@ Use variants to differentiate the hierarchy.
 
 <Story id="inputs-button-button--hierarchy" />
 
-### File upload
-Example of how to do file upload button. Please note this demo only works in Storybook Canvas (isolated example only).
+### Compact
+Compact mode is toggled using [EdsProvider](?path=/docs/edsprovider--introduction).
+
+Note that compact mode should come with the possibility to switch to normal mode for accessibility reasons.
 
-<Story id="inputs-button-button--file-upload" />
+<Story id="inputs-button-button--compact" />
 
 ### Progress button
 Use the `Progress` component if you need progress indication on a button.
 
 <Story id="inputs-button-button--progress-button" />
 
-### All
-Example All button combinations. 
+### With tooltip
+To display a tooltip on the `Button` you can wrap it in a `Tooltip`. If you need to disable the Button, the button should be disabled using
+`aria-disabled` instead of the `disabled` prop because Chrome doesn't expose the hover event on disabled buttons. 
```

Comment:
This is a bit too specific, and therefore not entirely correct. Tooltip also does not work in `edge`, and in `safari` there is an onmouseover event, but no onmouseleave event, so the tooltip gets stuck always on. Instead i suggest: 

Title: "Disabled buttons and tooltip"
text: "Due to inconsistent browser support for mouse events on disabled buttons, `Tooltip` should not be used in combination with `disabled` on `Button`. Instead use `aria-disabled` and manually block the button `onclick` action. This has the added benefit of making the `Tooltip` accessible for people using keyboard navigation, as the button is still focusable."